inputfiles/idl/fetch.commentmap.json

```diff
@@ -4,7 +4,7 @@
     "request-url": "Returns the URL of request as a string.",
     "request-headers": "Returns a Headers object consisting of the headers associated with request. Note that headers added in the network layer by the user agent will not be accounted for in this object, e.g., the \"Host\" header.",
     "request-destination": "Returns the kind of resource requested by request, e.g., \"document\" or \"script\".",
-    "request-referrer": "Returns the referrer of request. Its value can be a same-origin URL if explicitly set in init, the empty string to indicate no referrer, and \"about:client\" when defaulting to the global's default. This is used during fetching to determine the value of the `Referer` header of the request being made.",
+    "request-referrer": "Returns the referrer of request. Its value can be a same-origin URL if explicitly set in init, the empty string to indicate no referrer, and \"about:client\" when defaulting to the global's default. This is used during fetching to determine the value of the `Referrer` header of the request being made.",
```

Comment:
It's an infamous implementation level typo: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer

Unfortunately it's way too late to fix it in browsers, so we need to just deal with it.

Reply:
Oh, wow 😂